certbot/plugins/standalone_test.py

```diff
@@ -21,8 +25,8 @@ class ServerManagerTest(unittest.TestCase):
 
     def setUp(self):
         from certbot.plugins.standalone import ServerManager
-        self.certs = {}
-        self.http_01_resources = {}
+        self.certs = {}  # type: Dict[bytes, Tuple[OpenSSL.crypto.PKey, OpenSSL.crypto.PKey]]
+        self.http_01_resources = {}  # type: Set[challenges.HTTP01Response]
```

Comment:
Isn't this `Set[acme_standalone.HTTP01RequestHandler.HTTP01Resource]`?

Reply:
fixed